src/utils/parser.ts

```diff
@@ -32,4 +33,59 @@ function prepareParserConfig(req?: Request) {
   };
 }
 
-export { prepareParserConfig, parse };
+const TYPES_400 = [
+  'null-or-falsey-document',
+  'impossible-to-convert-to-json',
+  'invalid-document-type',
+  'invalid-json',
+  'invalid-yaml',
```

Comment:
It is not very ideal to depend on hardcoded strings instead of constants declared in Parser-JS package. I would suggest creating a follow-up issue so we move those strings to public constants in Parser-JS so we can use it here later.

Reply:
@smoya I created the issue in Parser-JS here: https://github.com/asyncapi/parser-js/issues/448 👍